Makefile

```diff
@@ -260,7 +260,27 @@ release-manager-snapshot: release
 .PHONY: release-manager-release
 release-manager-release: release
 
+JAVA_ATTACHER_VERSION:=1.26.0
+JAVA_ATTACHER_JAR:=apm-agent-attach-cli-$(JAVA_ATTACHER_VERSION).jar
+JAVA_ATTACHER_SIG:=$(JAVA_ATTACHER_JAR).asc
+JAVA_ATTACHER_URL:=https://search.maven.org/remotecontent?filepath=co/elastic/apm/apm-agent-attach-cli/$(JAVA_ATTACHER_VERSION)/$(JAVA_ATTACHER_JAR)
+JAVA_ATTACHER_SIG_URL:=https://search.maven.org/remotecontent?filepath=co/elastic/apm/apm-agent-attach-cli/$(JAVA_ATTACHER_VERSION)/$(JAVA_ATTACHER_SIG)
```

Comment:
Better to use `https://repo1.maven.org/maven2/co/elastic/apm/apm-agent-attach-cli` as the base URL. That makes it easier to add just maven central to an allow list when running in an air-gapped environment.